src/config/sentry.ts

```diff
@@ -1,8 +1,12 @@
 import * as Sentry from '@sentry/node';
+import { envVars } from './environment-variables';
+const { sentryServerDSN, microsEnv, microsServiceVersion, microsEnvDefault } =
+  envVars;
 
-export default (): void => {
-  Sentry.init({
-    dsn: process.env.SENTRY_DSN,
-    release: process.env.HEROKU_SLUG_COMMIT,
+export const initializeSentry = (): void => {
+  return Sentry.init({
+    dsn: sentryServerDSN,
+    environment: microsEnv || microsEnvDefault,
```

Comment:
Sorry, I might have not been clear.  Instead of having `microsEnv || microsEnvDefault` in multiple places (which we already do), we can just set `microsEnv: process.env.MICROS_ENV || "development"` in the env vars so we never have to deal with that logic again.